assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -450,17 +474,17 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
     .call(truncate, resourceLabelWidth)
 
   // Add links and icons to Y axis
-  d3.selectAll('.axis--y .tick').each(function (d) {
+  d3.selectAll('.axis--y .tick').each(function (d, i) {
```

Comment:
I don't see where the this new variable `i` is used?

Reply:
Hm, yeah, it's not, nice catch, I was using it before but my implementation of this changed a few times and this is no longer needed. I will remove.

Reply:
Done https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1472/commits/3b5e1823abb0dabba0bdab4cbd58c71c921946dd